std/uni.d

```diff
@@ -6968,18 +6968,22 @@ static assert(Grapheme.sizeof == size_t.sizeof*4);
         $(LREF icmp)
         $(REF cmp, std,algorithm,comparison)
 +/
-int sicmp(S1, S2)(S1 str1, S2 str2) if (isSomeString!S1 && isSomeString!S2)
+int sicmp(S1, S2)(S1 r1, S2 r2) @nogc nothrow
```

Comment:
Shouldn't add @nogc nothrow -- someone might want to use this where `S1` and `S2` can throw / allocate.